packaged/lisk.sh

```diff
@@ -285,6 +286,12 @@ start_lisk() {
 	fi
 }
 
+setLogspath() {
+	jq '.apps[0].out_file = "logs/\(env.LISK_NETWORK)/lisk.app.log"' < etc/pm2-lisk.json|jq '.apps[0].error_file = "logs/\(env.LISK_NETWORK)/lisk.app.err"' > tmp.json
```

Comment:
- use `$LISK_NETWORK` variable
- invoke `jq` only once
- use `mktemp`

```suggestion
	TEMP=$( mktemp )
	jq '.apps[0].out_file = "logs/'"$LISK_NETWORK"'/lisk.app.log"|.apps[0].error_file = "logs/'"$LISK_NETWORK"'/lisk.app.err"' "$PM2_CONFIG" >"$TEMP"
	mv -f "$TEMP" "$PM2_CONFIG"
```